plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeDelete.java

```diff
@@ -270,6 +270,19 @@ public void testDeleteMultiplePartitionKeys()
         assertQuery("SELECT * FROM " + tableName, "VALUES (1, 2, 4), (3, 2, 1), (null, null, null), (1, 1, 1)");
     }
 
+    // This is not the best class for this test but it doesn't fit any other places
+    // and here entire test infrastructure is present and ready
```

Comment:
This is a discourse between code author and a reviewer. Doesn't need to be persisted in the source code comment.

However, indeed, the class is about deletes and we don't do any deletes here.
let's move it elsewhere, perhaps into BaseDeltaLakeMinioConnectorTest

Reply:
I wanted to avoid the discussion and leave it for future maintainers ;)